Resources/Prototypes/Catalog/Fills/Lockers/misc.yml

```diff
@@ -135,6 +136,8 @@
           prob: 0.20
         - id: DrinkSpaceLube
           prob: 0.20
+        - id: ClothingMaskBalaclava
```

Comment:
I don't think this should be available in maints loot.

Reply:
i dont want people to be valid just for wearing it or even have it be emag only, its intended to be a somewhat abundant and free clothing item just to conceal your identity and for space russian roleplay